### ISSUE TITLE
Tags URLS are Not slugified in 1.10.3

### ISSUE THREAD

**Issue by mskian** [lumeland/lume#239]
After Updating to `v1.10.3` tag URLs are not slugified 
in `v1.10.1` it slugify the tag URL paths 

Command - `deno task build`

expected output - /tags/hello-world/
actual output - /tag/hello world/

**Comment by mskian** [comment]
using this solution to fix the slugify for tags

Template - `tag.tmpl.js`

```js
export const layout = "layouts/tag.njk";

export default function* ({ search }) {
  for (const tag of search.tags()) {
    yield {
      url: `/tag/${tag.toLowerCase().replaceAll(" ", "-")}/`,
      title: `Tagged - ${tag}`,
      description: `Tagged - ${tag}.`,
      type: "tag",
      tag,
    };
  }
}
```

**Comment by oscarotero** [comment]
Thanks. This is a bug. `v1.10.2` changed the execution time of the preprocessors and I missed this. Sorry, I'll fix it.
Meanwhile, you can use the `slugify` helper, provided by the slugifyUrls plugin:

```ts
export default function* ({ search }, { slugify }) {
  for (const tag of search.tags()) {
    yield {
      url: slugify(`/tag/${tag}/`),
      title: `Tagged - ${tag}`,
      description: `Tagged - ${tag}.`,
      type: "tag",
      tag,
    };
  }
}
```

**Comment by mskian** [comment]
Welcome :-) 
Thanks for the Quick response
it's great to know `slugifyUrls` about helper

**Comment by oscarotero** [comment]
Fixed in v1.10.4
